### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by tambry** [moonlightwork/bug-tracker#9]
The Moonlight website is inaccessible over IPv6.

### What are the steps to reproduce the issue?

1. Try loading the website with only IPv6 connectivity available.

**Comment by philipithomas** [comment]
@tambry Oh no! What do you get if you do `curl https://www.moonlightwork.com`? Is there an error message shown?

I think I've been on IPv6 addresses on my phone and not had problems, so any details you can provide would help to get this fixed.

**Comment by tambry** [comment]
```
./curl.exe https://www.moonlightwork.com
curl: (6) Could not resolve host: www.moonlightwork.com
```

The problem only occurs when there's **only** IPv6 connectivity (no IPv4 connectivity). You can force curl to use IPv6 with the `-6` flag (e.g. `curl https://www.moonlightwork.com -6`).

**Comment by philipithomas** [comment]
![screenshot_20180414-145602](https://user-images.githubusercontent.com/1312414/38771609-2646575a-3ff4-11e8-92a3-bfcd4142a865.png)
![screenshot_20180414-145656](https://user-images.githubusercontent.com/1312414/38771610-2a0c3918-3ff4-11e8-9ec1-ba4c89f47210.png

I'm on IPv6 on my phone and it seems to be working. Are you sure it's IPv6?

**Comment by philipithomas** [comment]
The load balancer is Google Cloud HTTP load balancer, which supports IPv6. If there were an application error, I would expect the LB to throw a 503 or something, not refuse the connection. https://cloud.google.com/compute/docs/load-balancing/ipv6

**Comment by philipithomas** [comment]
Ah, looks like we need AAAA records. I'll update when the DNS changes have propagated.

**Comment by tambry** [comment]
The issue manifests only when there's **_only_** IPv6 connectivity and no IPv4 support. Note that the website lacks an AAAA DNS record and only has an A record. Thus when someone tries to connect over IPv6 with no IPv4 fallback available, there's no address to connect to.

```
C:\Users\Raul>nslookup www.moonlightwork.com
Server:  anycast.censurfridns.dk
Address:  2001:67c:28a4::

Nicht autorisierende Antwort:
Name:    www.moonlightwork.com
Address:  35.190.34.154
```

**Comment by philipithomas** [comment]
I think it's fixed!

Validation seems to be passing: http://ipv6-test.com/validate.php

Can you confirm that it's working for you @tambry ?

**Comment by tambry** [comment]
@philipithomas That's unfortunate to hear! They seem to be using AWS EC2, so fixing this should be a matter of someone clicking around in the control panel for a few minutes (unless their instances are really old). Hopefully they won't be left too far behind in the dust before they do fix this.

Once again, thanks for looking into all of this. :)

**Comment by philipithomas** [comment]
@tambry Yes, I agree - but it's possible that their load balancers, rate limiters, or audit logs don't support ipv6 👎
